sphinxarg/ext.py

```diff
@@ -10,6 +10,8 @@
 from docutils.statemachine import StringList
 from docutils.utils import new_document
 from sphinx.util.nodes import nested_parse_with_titles
+from sphinx.ext.autodoc import mock
```

Comment:
Could this import ever fail? I forget if autodoc is shipped with Sphinx or an extra module that needs to be installed.